examples/flutter_gallery/lib/gallery/app.dart

```diff
@@ -56,11 +61,11 @@ class GalleryAppState extends State<GalleryApp> {
         });
       },
       showPerformanceOverlay: _showPerformanceOverlay,
-      onShowPerformanceOverlayChanged: (bool value) {
+      onShowPerformanceOverlayChanged: config.enablePerformanceOverlay ? (bool value) {
```

Comment:
This shouldn't be needed. If we're not adding the performance overlay drawer item then there's no risk of it being called.